src/katsdpnetboxutilities/system/make_config/makeconfig.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+
+import sys
+import yaml
+
+def make_header(NAME):
+    header_file = ["usepackage{fancyhdr}", "pagestyle{fancy}", "fancyhf{}"]
+    header_file.append("rhead{\\textbf{%s}}" % NAME)
+    with open("/reports/header.tex", "w+") as f:
+        for txt in header_file:
+            f.write("\\" + txt + "\n")
+        f.close()
+
+def make_pandoc_yaml(NAME):
+    config_dict = {
+      "standalone": True,
+      "self-contained": True,
+      "variables": {
+        "documentclass": "book",
```

Comment:
documentclass=report should give a bit more of acondensed file.

Reply:
article is the other option for documentclass. Try out report and article and see if they are better. I did a quick test on report and I think its nicer.